src/utils/es_query/from_kuery.js

```diff
@@ -17,33 +17,20 @@
  * under the License.
  */
 
-import { fromLegacyKueryExpression, fromKueryExpression, toElasticsearchQuery, nodeTypes } from '../../../kuery';
-import { documentationLinks } from '../../../documentation_links';
-import { NoLeadingWildcardsError } from '../../../kuery/errors';
-
-const queryDocs = documentationLinks.query;
+import { fromLegacyKueryExpression, fromKueryExpression, toElasticsearchQuery, nodeTypes } from '../kuery';
 
 export function buildQueryFromKuery(indexPattern, queries = [], config) {
   const allowLeadingWildcards = config.get('query:allowLeadingWildcards');
-
-  const queryASTs = queries.map((query) => {
+  const queryASTs = queries.map(query => {
     try {
       return fromKueryExpression(query.query, { allowLeadingWildcards });
-    }
-    catch (parseError) {
-      if (parseError instanceof NoLeadingWildcardsError) {
-        throw parseError;
-      }
-
+    } catch (parseError) {
       try {
         fromLegacyKueryExpression(query.query);
-      }
-      catch (legacyParseError) {
+      } catch (legacyParseError) {
         throw parseError;
       }
-      throw new Error(
-        `It looks like you're using an outdated Kuery syntax. See what changed in the [docs](${queryDocs.kueryQuerySyntax})!`
```

Comment:
Why did you remove the link to the docs? Without any direction I think it'll be impossible for a user to find the right information.

Reply:
I didn't want it to rely on something in `public/` (which is where `documentationLinks` is). I guess I could just make the invoking function in `public/` check for this type of error and add the links.

Reply:
The documentation_links module could also probably be moved to utils?

Reply:
That module relies on ui/metadata which I'm not sure could be moved to utils...

Reply:
It only uses the branch name though. I think the stuff in `metadata` ultimately comes from the server, so the branch name should be accessible somewhere.